app/views/tracks/new.html.erb

```diff
@@ -73,8 +76,6 @@
   }
 
   var rhomb = new Rhombus();
-  var ptnId = rhomb._song.addPattern();
```

Comment:
Before, there was a single pattern which was created automatically when rhomb was instantiated. Now, patterns need to be explicitly added by the music application.